services/frontend/src/scenes/Projects/Projects.scss

```diff
@@ -0,0 +1,115 @@
+@import "styles/_colors.scss";
+@import "styles/_dimens.scss";
+@import "styles/_mixins.scss";
+
+.projects {
+    display: flex;
+    flex-direction: column;
+    height: calc(100% - #{$navbar-height});
+    overflow: auto;
+}
+
+.projects-header {
+    border-bottom: 1px solid $color-text-secondary;
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    padding: $default-spacing;
+}
+
+.filter-button {
+    background-color: transparent;
+    border: none;
+    border-radius: $default-border-radius;
+    box-shadow: none;
+    color: $color-text-secondary;
+    margin: 0 16px;
+    padding: $default-spacing/2 $default-spacing*1.5;
+    text-transform: none;
+
+    &.filter-button--active {
+        background-color: $color-primary;
+        color: $color-light;
+    }
+}
+
+.projects-list {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    padding: $default-spacing * 2;
+
+    .project-card {
+        margin: $default-spacing 0;
```

Comment:
I can see why this would be nested in `.projects-list` however would it be better to keep the css all in one place and move this to the `.project-card` selector below?

Reply:
It would, if I didn't plan on eventually moving the `ProjectCard` out into its own component file. Since the margin is really only relevant from the point-of-view of the list, I think it makes more sense to scope it with the list (as opposed to making it a default margin value for all ProjectCards, and then having to override it for later uses).

Reply:
Yeah, it make's sense to scope within the list logically. I'm alright with it either way and if you plan to extract that into it's own component it's good for now.